inc/Theme.php

```diff
@@ -19,7 +19,7 @@
     // "comment-form" // until someone makes the hard coded novalidate attr filterable this stays off.
   ]);
 
-  $GLOBALS["content_width"] = 800; // Width of the actual content in single page.
+  $GLOBALS["content_width"] = 2560;
```

Comment:
So this makes things like YouTube embeds 2560 pixels wide. I don't know if that's a problem, just saying.

Reply:
Yup, this is acknowledged and it's "fixed" by wrapping all oEmbeds within the responsive-embed wrapper.